geonode/documents/templates/documents/document_upload_base.html

```diff
@@ -10,6 +10,7 @@
 <div class="page-header">
   <a href="{% url "document_browse" %}" class="btn btn-primary pull-right">{% trans "Explore Documents" %}</a>
   <h2 class="page-title">{% trans "Upload Documents" %}</h2>
+  <p>Allowed document types [doc, docx, gif, jpg, jpeg, ods, odt, odp, pdf, png, ppt, pptx, rar, tif, tiff, txt, xls, xlsx, xml, zip, gz]
```

Comment:
I was thinking the same, but I don't think there is a better way other than setting this in a context variable or from context processor.
Does such an overhead justifies the effort?